examples/project_ask_ai_dagster/project_ask_ai_dagster/resources/pinecone.py

```diff
@@ -0,0 +1,44 @@
+from typing import Optional
+
+import dagster as dg
+from langchain_openai import OpenAIEmbeddings
+from pinecone import Pinecone, ServerlessSpec
+from pydantic import Field, PrivateAttr
+
+
+class PineconeResource(dg.ConfigurableResource):
+    pinecone_api_key: str = Field(description="Pinecone API key")
+    openai_api_key: str = Field(description="OpenAI API key")
+    pinecone_env: str = Field(description="Pinecone environment")
```

Comment:
It doesn't seem like `pinecone_env` is used in this resource. I also though Pinecone just relied on the API key